vela-templates/definitions/internal/labels.cue

```diff
@@ -9,9 +9,18 @@ labels: {
 	}
 }
 template: {
-	patch: spec: template: metadata: labels: {
-		for k, v in parameter {
-			"\(k)": v
+	patch: {
+		metadata: {
+			labels: {
+				for k, v in parameter {
+					"\(k)": v
+  			}
```

Comment:
Seems here is not well aligned. I would recommend a format fix.

Reply:
thanks @Somefive , I have fixed the formatting.